qml/main.qml

```diff
@@ -42,6 +42,7 @@ Window {
     }
 
     StackView {
+        property int mccOffsetFromUTC: -1
```

Comment:
How do I do this?

I also thought it was out of place here, but had no idea how to solve 
this differently.

On 10.02.2015 08:18, Simon Busch wrote:
> In qml/main.qml [1]:
> 
>> @@ -42,6 +42,7 @@ Window {
>> }
>> 
>> StackView {
>> + property int mccOffsetFromUTC: -1
> 
> Why do we have this here? If you need to share things between two
> components use a static js library.
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1]
> https://github.com/webOS-ports/org.webosports.app.firstuse/pull/7#discussion_r24393217
> [2]
> https://github.com/webOS-ports/org.webosports.app.firstuse/pull/7/files#r24393217